### ISSUE TITLE
Clarification of the way to open the MDwiki site locally by FireFox. 

### ISSUE THREAD

**Issue by hiroTochigi** [open-learning-exchange/open-learning-exchange.github.io#2437]
### Problem
When you look at the OLE Support site to check your correction in your local environment, you need to open the website from Index.html by your browser. The link explains how to look at MDwiki page locally, but it is not clear enough for a layman to understand how to look at the page.   

### Steps to reproduce the problem

Try to open the page by file:///.../username.github.io/mdwiki.html#!index.md.

### Screenshots

N/A

### Proposed solution

It is clearer to explain how to open the website locally below.

Go to your Username.github.io directory and open Index.html by **FireFox**. You can see almost all expression of MDwiki in this method. 
If your OS is Mac or Linux, you can see the page via localhost by executing the below command in the same directory.
```
python -m SimpleHTTPServer
```
Then, open a web browser and type localhost:8000 in the address bar.
You can read further information in [this link](http://dynalon.github.io/mdwiki/#!faq.md).
Also, you can see the general Markdown expression on [this website](https://dillinger.io/).

**Comment by ArranHL** [comment]
Hi @hiroTochigi, are you referring here to the Preview Changes Locally section of the [GitHub Issues step](https://open-learning-exchange.github.io/#!pages/vi/vi-github-issues.md) ?

**Comment by hiroTochigi** [comment]
@ArranHL, Yes, I do.

**Comment by irisb1701** [comment]
I agree the link is a little vague. What command would you use to "open index.html by Firefox"?
